README.md

```diff
@@ -2,11 +2,18 @@
 
 ## A pre-populated demo of the OCS services
 
-This example project provides a docker-compose file which launches an [Observation Portal](https://github.com/observatorycontrolsystem/observation-portal), a [web interface](https://github.com/observatorycontrolsystem/ocs-example-frontend) for the observation portal, a [Configuration Database](https://github.com/observatorycontrolsystem/configdb), a [Downtime Database](https://github.com/observatorycontrolsystem/downtime), and an [Adaptive Scheduler](https://github.com/observatorycontrolsystem/adaptive_scheduler), all connected together with pre-populated sample data.
+This example project provides a docker-compose file which launches 
+an [Observation Portal](https://github.com/observatorycontrolsystem/observation-portal), 
+a [web interface](https://github.com/observatorycontrolsystem/ocs-example-frontend) for the observation portal, 
+a [Configuration Database](https://github.com/observatorycontrolsystem/configdb), 
+a [Downtime Database](https://github.com/observatorycontrolsystem/downtime), 
+a [Science Archive](https://github.com/observatorycontrolsystem/science-archive), 
+and an [Adaptive Scheduler](https://github.com/observatorycontrolsystem/adaptive_scheduler), 
+all connected together with pre-populated sample data.
 
 ## Credentials
 
-By default, the Observation Portal will have a single account created with the username `test_user` and the password `test_pass`. This can be used to login to the Observation Portal or the admin interface of the Configuration or Downtime Databases.
+By default, the Observation Portal will have a single account created with the username `test_user` and the password `test_pass`. This can be used to login to the Observation Portal, Science Archive, or the admin interface of the Configuration or Downtime Databases.
```

Comment:
It's the admin interface of the science archive that you are logging into right? So I would just put "Science Archive" along with "Configuration and Downtime Databases"